node/node.go

```diff
@@ -18,6 +20,7 @@ var Model = resource.NewModel("viam", "lorawan", "node")
 type Config struct {
 	JoinType    string `json:"join_type,omitempty"`
 	DecoderPath string `json:"decoder_path"`
+	Interval    string `json:"uplink_interval_mins"`
```

Comment:
actually why not make this a float? seems weird as a string

Reply:
fine with changing to a float but its a string in the scope

Reply:
I'm okay with changing a type of attribute from the scope doc if it makes more sense, + there was no big motivating reason to have it be a string anyway.

Reply:
sounds good will change!